.gitignore

```diff
@@ -1,3 +1,9 @@
+# For dev purposes
+docker-compose.yml
```

Comment:
why do we have docker-compose.yml in gitignore? This may cause issues later on when we do actually use docker-compose

Reply:
I have a docker-compose that starts up an instance of mongodb and mongo express for dev purposes.